app/views/users/settings.html.erb

```diff
@@ -25,41 +31,43 @@
 <% end %>
 
 <% if allowed_to?(:manage?, KeyPair) %>
-  <h2><%= t('key_pairs.key_pairs_title') %></h2>
+  <p>
```

Comment:
Revert this change (starting from this line and extending below). There's more appropriate ways of adding some whitspace, if that's what you were trying to do, and either way it shouldn't be in this PR.